parser-typechecker/src/Unison/Hashing/V2/Convert.hs

```diff
@@ -347,44 +347,47 @@ m2hPatch (Memory.Patch.Patch termEdits typeEdits) =
       Memory.TypeEdit.Deprecate -> Hashing.TypeEdit.Deprecate
 
 hashPatch :: Memory.Patch.Patch -> Hash
-hashPatch = H.accumulate' . m2hPatch
+hashPatch = Hashing.Patch.hashPatch . m2hPatch
 
-tokensBranch0 :: Accumulate h => Memory.Branch.Branch0 m -> [Token h]
-tokensBranch0 = H.tokens . m2hBranch
+hashBranch0 :: Memory.Branch.Branch0 m -> Hash
+hashBranch0 = Hashing.Branch.hashBranch . m2hBranch0
 
 -- hashing of branches isn't currently delegated here, because it's enmeshed
```

Comment:
Are we planning to change this at all as part of this migration?

Reply:
Oops, yes, this PR changes this.